arnon_data_prep.Rmd

```diff
@@ -70,7 +73,12 @@ readr::write_tsv(prepped.data, file.path("arnon_data", "normalized_arnon",
 readr::write_tsv(data.frame(batch.info), file.path("arnon_data",
                                                    "batch.info.txt"))
 ```
-
+### Write metadata info to be used later
+```{r Write batch file}
+# Write metadata file for subsequent testing
+geo.meta %>% dplyr::select("cell.type.ch1", "characteristics_ch1.2") %>%
```

Comment:
Difference from what you have earlier is that you're only saving selected metadata, do I have that right?

Reply:
Yes. Saving only the metadata we want to test/label